package.json

```diff
@@ -46,11 +46,9 @@
     "test.dist": "node scripts/build --validate-build",
     "test.end-to-end": "cd test/end-to-end && npm ci && npm test && npm run test.dist",
     "test.jest": "node --experimental-vm-modules ./node_modules/jest/bin/jest.js",
-    "test.karma": "cd test/karma && npm ci && npm run karma",
-    "test.karma.prod": "cd test/karma && npm ci && npm run karma.prod",
     "test.wdio": "cd test/wdio && npm ci && npm run test",
     "test.wdio.testOnly": "cd test/wdio && npm ci && npm run wdio",
-    "test.prod": "npm run test.dist && npm run test.end-to-end && npm run test.jest && npm run test.karma && npm run test.sys.node && npm run test.testing && npm run test.analysis",
+    "test.prod": "npm run test.dist && npm run test.end-to-end && npm run test.jest && npm run test.sys.node && npm run test.testing && npm run test.analysis",
```

Comment:
Can we add `test.wdio` to this script?

Reply:
sure, makes sense

Reply:
I don't know if we use this script anywhere though! I kinda forgot it existed